dts/arm/nordic/nrf51822_qfaa.dtsi

```diff
@@ -14,3 +14,9 @@
 &sram0 {
 	reg = <0x20000000 DT_SIZE_K(16)>;
 };
+
+/ {
+	soc {
+		compatible = "nrf,nRF51822-QFAA", "nrf,nRF51822", "nrf,nRF51", "simple-bus";
```

Comment:
Shouldn't this be rather "nordic," instead of "nrf,"?

Reply:
'nrf,` is the nordic dts vendor prefix that's what we use everywhere else.

Reply:
Pehaps I got lost, but I thought it was exactly "nordic," that was used as the vendor prefix. Like in "nordic,nrf-i2c", "nordic,nrf-spi", or "nordic,nrf52-flash-controller".

Reply:
ah you are correct, looks like there is a stray 'nrf,' for some flash controller props.